commons-component-common/src/main/java/org/exoplatform/commons/notification/lifecycle/MailLifecycle.java

```
@@ -120,7 +120,11 @@ public void send(NotificationContext ctx) {
       MessageInfo msg = builder.buildMessage(ctx);
       if (msg != null) {
         if (NotificationUtils.isValidEmailAddresses(msg.getTo()) == true) {
-          CommonsUtils.getService(QueueMessageImpl.class).sendMessage(msg.makeEmailNotification());
+          try {
+            CommonsUtils.getService(QueueMessage.class).sendMessage(msg.makeEmailNotification());
+          } catch (Exception e) {
+            //error in sending message
```

You should log this error at least (no exception swallow plz)